### PR TITLE
Fix inadvertent plugin/theme volume deletion

### LINKED ISSUE #102: My Themes and Plugins get pruned despite [local] prefix
## Overview

- What's your problem about?
    - Local themes & plugins are pruned / deleted 
- What is your operating system?
    - Mac OSX Sierra 10.12.5
- What is your docker version? `docker version`
    - 17.03.1-ce

## Details

When I run `docker-compose up -d && docker-compose logs -f wordpress` the contents of `wp-content/themes/Divi` is deleted. The folder remains, but no files are left. It reports `Warning: Could not fully remove the theme Divi.`. 

I read through this [issue](https://github.com/visiblevc/wordpress-starter/issues/84) and followed the recommendations there. As you can see in my yml file, I've got it added to the theme list like so: `[local]Divi`

What am I missing?

## `docker-compose.yml`

```yml
version: '2'
services:
  wordpress:
    image: visiblevc/wordpress:latest
    ports:
      - 8080:80
    volumes:
      # database configuration
      - ./data:/data
      # development theme setup
      - ./wp-content/themes/Divi:/app/wp-content/themes/Divi
      # WordPress uploads folder
      - ./wp-content/uploads:/app/wp-content/uploads
    environment:
      DB_NAME: wordpress
      DB_PASS: root
      PLUGINS: >-
        wordpress-importer,
      THEMES: >-
        [local]Divi,
      WP_DEBUG: 'true'
  db:
    image: mariadb:10 # or mysql:5.7
    volumes:
      - data:/var/lib/mysql
    environment:
      MYSQL_ROOT_PASSWORD: root
  phpmyadmin:
    image: phpmyadmin/phpmyadmin
    environment:
      MYSQL_ROOT_PASSWORD: root
    ports:
      - 22222:80
volumes:
  data:
```

## Project structure

```
/data
  /database.sql
/wp-content
  /themes
    /Divi
```

## `docker-compose up` output

```
wordpress_1   | ==> Checking themes
wordpress_1   | Warning: Could not fully remove the theme Divi.
wordpress_1   | Error: No themes deleted.
wordpress_1   | ==> Checking plugins
wordpress_1   | Success: Plugins already activated.
wordpress_1   | ==> Finalizing
```

### PR DESCRIPTION
**Pull request by dsifford**
cc: @karellm

This PR _really_ fixes this issue for real this time. I tested it on both linux and OSX and it seems to work fine in all cases.

Closes #102

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,7 +16,6 @@ Thumbs.db
 
 # Wordpress
 wp-content/*
-!wp-content/themes/
```
Comment (karellm) [inline .gitignore]: The idea of this was to whitelist the themes you want. Is there a reason you removed it?

Comment (dsifford) [inline .gitignore]: Oh, whoops! I added a test `wp-content` directory at the top level of the repo for testing plugins and themes and whitelisting the themes directory made the test theme that I was using visible to this git repo.

Do you think we should add another gitignore file in the example directory?

Comment (dsifford) [inline .gitignore]: Just pushed a commit that adds an example `.gitignore` as well as a direct download one-liner to get the example directory only.

**Comment by karellm** [comment]
LGTM 👍

**Pull request merged**